full-service/src/service/webhook.rs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2018-2024 MobileCoin Inc.
+
+//! Manages sending a webhook for synced accounts that have received deposits
```

Comment:
🆕 Moved this to its own file - this is the main change: introduces the WebhookThread